README.md

```diff
@@ -25,14 +25,33 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+    - name: Delete Old Artifacts
+      uses: actions/github-script@v6
+      id: artifact
+      with:
+        script: |
+          const res = await github.rest.actions.listArtifactsForRepo({
+            owner: context.repo.owner,
```

Comment:
Checking in.. Wait, we don't need to pass any GH_ACTION token?

Reply:
Looks like GitHub creates one automatically: https://docs.github.com/en/actions/security-guides/automatic-token-authentication